sdk/servicebus/Microsoft.Azure.WebJobs.Extensions.ServiceBus/src/Triggers/DataContractBinarySerializer.cs

```diff
@@ -0,0 +1,121 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.IO;
+using System.Runtime.Serialization;
+using System.Xml;
+
+namespace Microsoft.Azure.WebJobs.Extensions.ServiceBus.Triggers
```

Comment:
This is ported from the T1 SB SDK to support deserializing XML payloads. It was not included in the T2 SB SDK, so it is included here.